src/coreclr/pal/src/exception/remote-unwind.cpp

```diff
@@ -917,35 +916,6 @@ ExtractProcInfoFromFde(
         return false;
     }
 
-    // Now fill out the proc info if requested
```

Comment:
HAVE_GET_PROC_INFO_IN_RANGE is not defined on Unix when libunwind lacks support for unw_get_proc_info_in_range. This function is only available in HP libunwind version 1.6 and later. To ensure compatibility, we need to handle this for LLVM libunwind and HP libunwind versions earlier than 1.6.

Reply:
That's a fundamental misunderstanding of what this PR does (or I am missing something).

This code is Apple specific. It was called explicitly on Apple platforms already without any libunwind involvement (aside from operating on the same structures). I completely removed the code path that goes to (HP) libunwind on Apple platforms because there are only two unwinding encodings (compact unwinding and DWARF; with the DWARF always using compact unwinding as index because the linker enforces that) and both are already handled by the special Apple code path.

Reply:
ExtractProcInfoFromFde is not Apple specific.  Yes, most of the calls are in Apple specific except here: https://github.com/dotnet/runtime/blob/342936c5a88653f0f622e9d6cb727a0e59279b31/src/coreclr/pal/src/exception/remote-unwind.cpp#L2398

Reply:
Hmm. I think that code path never takes place with the HP libunwind fork that we ship. It may be relevant for platforms that use system libunwind (FreeBSD or Haiku). I can add that code path back...

Reply:
Yes, add it back to make sure ever platform is covered.